packages/image_picker/lib/image_picker.dart

```diff
@@ -52,7 +52,7 @@ class ImagePicker {
         'maxHeight': maxHeight,
       },
     );
-
+    print("image picker : "+path);
```

Comment:
This code doesn't look relevant to fingerprint functionality. Is this change here intentionally?